deployment/mysql/dbscripts/init.sql

```diff
@@ -36,7 +36,7 @@ CREATE TABLE IF NOT EXISTS REGISTRY_ORGANIZATION (
     ORG_NAME                 VARCHAR(255)               NOT NULL,
     DESCRIPTION              VARCHAR(255)               DEFAULT "",
     WEBSITE_URL              VARCHAR(255)               DEFAULT "",
-    DEFAULT_IMAGE_VISIBILITY ENUM("PUBLIC", "PRIVATE")  NOT NULL DEFAULT "PUBLIC",
+    DEFAULT_VISIBILITY       ENUM("PUBLIC", "PRIVATE")  NOT NULL DEFAULT "PUBLIC",
```

Comment:
Let's not change this. This is the image's visibility. Not the organization's visibility.